### DISCUSSION TITLE
Undo Queue with mouse

### DISCUSSION THREAD

**Discussion by madprops** [Taiko2k/Tauon#1194] [Ideas]
It's great that we can queue by middle clicking a song, but to undo that we have to open the panel on the left and remove the items by middle clicking them. I have 2 ideas regarding this.

1) Setting so middle clicking a queued song un-queues it instead of queuing it multiple times (I would use this)

2) Ctrl or Shift + Middle Click does unqueue

**Comment by Solarunit** [comment]
1. I personally like to queue song multiple times with middle clicking
2. I agree,  would be nice to have. `Ctrl + Middle Click`
